disperser/dataapi/docs/swagger.yaml

```diff
@@ -293,6 +304,35 @@ paths:
       summary: Fetch throughput time series
       tags:
       - Metrics
+  /metrics/unsigned_batches:
```

Comment:
Here and other relevant locations: can we name it as `/metrics/operator_nonsigning_percentage`? 
In this API, we are interested in learning what's the nonsigning percentage for each operator.